dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -368,12 +368,90 @@
     </Style>
 
     <Style x:Key="NavigationViewPaneSearchButtonStyle" TargetType="Button">
-        <Setter Property="MinHeight" Value="40"/>
-        <Setter Property="Content" Value="&#xE11A;"/>
         <Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}" />
         <Setter Property="BorderThickness" Value="{ThemeResource NavigationViewToggleBorderThickness}" />
         <Setter Property="Background" Value="{ThemeResource NavigationViewItemBackground}" />
         <Setter Property="Foreground" Value="{ThemeResource NavigationViewItemForeground}" />
+        <Setter Property="Content" Value="&#xE11A;" />
+        <Setter Property="MinHeight" Value="36" />
+        <Setter Property="FocusVisualMargin" Value="0" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Button">
```

Comment:
This is so that the PointerOver and Pressed states will be aligned with the other NavView buttons. By default, the button style turns a lighter colour on PointerOver with the newest visual update.

Reply:
In that case, can't we just locally override the ButtonBackgroundPointerOver resource?

Reply:
The `Margin="4,2"` is also needed in the ContentPresenter element. I just tried prototyping it with Margin set on the Button itself. The focus rect turns out smaller, and I'm unable to adjust it with FocusVisualMargin?
![image](https://user-images.githubusercontent.com/7976322/111845426-0a228700-88c2-11eb-9117-228c676b7776.png)